app/common/src/appConfig.js

```diff
@@ -44,7 +44,7 @@ export async function readEnvironmentFromFile() {
     if (!isProduction || entries.length > 0) {
       Object.assign(process.env, variables)
     }
-    process.env.ENSO_CLOUD_DASHBOARD_VERSION ??= buildInfo.version
+    process.env.ENSO_CLOUD_DASHBOARD_VERSION ??= buildInfo.version ?? '0.0.1-dev'
```

Comment:
Absolutely tiny nitpick: Why `0.0.1`? I'm rather used to `0.0.0` or `0.1.0` as the latter may be shortened to just 0.1 (you don't actually expect bugfix version in unstable release).

Reply:
That's a mistake, should indeed be `0.0.0-dev`.